service/src/test/resources/ehr/mapper/condition/expected_output_linkset_1.xml

```diff
@@ -8,22 +8,22 @@
         <availabilityTime value="20200907101202" />
         <conditionNamed typeCode="NAME" inversionInd="true">
             <namedStatementRef classCode="OBS" moodCode="EVN">
-                <id root="7E277DF1-6F1C-47CD-84F7-E9B7BF4105DB"/>
+                <id root="Condition/7E277DF1-6F1C-47CD-84F7-E9B7BF4105DB"/>
```

Comment:
I know it is just the matter of mock, but it will be misleading for somebody reading that test, that the xml contains fhir reference. I think it is good idea to solve mocking mapperId the way you proposed. Maybe we could just change the id to not look like fhir reference, for example moving resource name at the end? 
` 7E277DF1-6F1C-47CD-84F7-E9B7BF4105DB-Condition `

Reply:
yes, it's very good idea to mock it your way, we really needed that, now we can see if the ids are correctly mapped!

Reply:
7E277DF1-6F1C-47CD-84F7-E9B7BF4105DB-Condition is actually kind of how EMIS generate their FHIR identifiers.

This is not, however, valid HL7 which requires real UUIDs. Maybe ok if we make it obvious we are using intentionally invalid identifiers? Related tech debt https://gpitbjss.atlassian.net/browse/NIAD-1293

Reply:
if we use the same id for each resource we won't see if there would be an error with for example assigning wrong reference or assigning the same id twice, especially in a huge bundle.

If that is emis identifier we could enrich the name with "test" or "mapper" or "new" suffix, for example 7E277DF1-6F1C-47CD-84F7-E9B7BF4105DB-Condition-new-id ?

Reply:
Any decision on this, @msolecka @mattd-kainos ?

Reply:
we can go with 7E277DF1-6F1C-47CD-84F7-E9B7BF4105DB-Condition-new-id

Reply:
I agree here that this convention of using "ResourceType/UUID" will confuse in the future because it looks like a FHIR reference. A suffix as suggested by Gosia maintains this traceability through the test fixtures without making the value appear to be something it is not.

We begin to get a bit verbose now but the information we're trying to convey is essentially `7E277DF1-6F1C-47CD-84F7-E9B7BF4105DB-was-generated-for-Condition`.

Note this still is not a valid HL7 instance identifier (II) but that is ok since these are mocks.

Reply:
Substituted `root="([^"/]+)/([^"]+)"` with `root="$2-$1-new-ID"` in all expected_output_linkset_*.xml files